VK_V1.0/VK_V1.0/Model/Photo/PhotoInfoDto.swift

```diff
@@ -10,4 +10,16 @@ final class PhotoInfoDto: Object, Decodable {
     @objc dynamic var url: String = ""
     /// Тип фото
     @objc dynamic var type: String = ""
+    /// Высота фото
+    var height: Int
+    /// Ширина фото
+    var width: Int
+    /// Соотношение сторон у фотографии
+    var aspectRatio: CGFloat {
+        CGFloat(height) / CGFloat(width)
```

Comment:
```suggestion
        CGFloat(height) / CGFloat(width)
```
не защиты от деления на 0